Content.Shared/Preferences/Appearance/HairStyles.cs

```diff
@@ -0,0 +1,189 @@
+            {"Rows Alt", "rows2"},
+            {"Rows Bun", "rows3"},
+            {"Flat Top", "sargeant"},
+            {"Scully", "scully"},
+            {"Shaved Mohawk", "shavedmohawk"},
+            {"Shaved Part", "shavedpart"},
+            {"Short Hair", "short"},
+            {"Short Hair 2", "short2"},
+            {"Short Hair 3", "short3"},
+            {"Short Bangs", "shortbangs"},
+            {"Overeye Short", "shortovereye"},
+            {"Shoulder-length Hair", "shoulderlen"},
+            {"Sidepart Hair", "sidepart"},
+            {"Side Ponytail", "stail"},
+            {"One Shoulder", "oneshoulder"},
+            {"Tress Shoulder", "tressshoulder"},
+            {"Side Ponytail 2", "ponytailf"},
+            {"Side Swipe", "sideswipe"},
+            {"Skinhead", "skinhead"},
+            {"Messy Hair", "smessy"},
+            {"Sleeze", "sleeze"},
+            {"Spiky", "spikey"},
+            {"Stylo", "stylo"},
+            {"Spiky Ponytail", "spikyponytail"},
+            {"Top Knot", "topknot"},
+            {"Thinning", "thinning"},
+            {"Thinning Rear", "thinningrear"},
+            {"Thinning Front", "thinningfront"},
+            {"Undercut", "undercut"},
+            {"Unkept", "unkept"},
+            {"Updo", "updo"},
+            {"Vegeta", "toriyama2"},
+            {"Overeye Very Short", "veryshortovereye"},
+            {"Overeye Very Short, Alternate", "veryshortovereyealternate"},
+            {"Volaju", "volaju"},
+            {"Wisp", "wisp"},
+            {"Zieglertail", "ziegler"},
+            {"Zone Braid", "zone"},
+        };
+
+        public static readonly Dictionary<string, string> FacialHairStylesMap = new Dictionary<string, string>()
+        {
+            {"Shaved", "shaved"},
+            {"Watson Mustache", "watson"},
+            {"Hulk Hogan Mustache", "hogan"},
+            {"Van Dyke Mustache", "vandyke"},
+            {"Square Mustache", "chaplin"},
+            {"Selleck Mustache", "selleck"},
+            {"Neckbeard", "neckbeard"},
+            {"Full Beard", "fullbeard"},
+            {"Long Beard", "longbeard"},
+            {"Very Long Beard", "wise"},
+            {"Elvis Sideburns", "elvis"},
+            {"Abraham Lincoln Beard", "abe"},
+            {"Chinstrap", "chin"},
+            {"Hipster Beard", "hip"},
+            {"Goatee", "gt"},
+            {"Adam Jensen Beard", "jensen"},
+            {"Volaju", "volaju"},
+            {"Dwarf Beard", "dwarf"},
+            {"3 O'clock Shadow", "3oclock"},
+            {"3 O'clock Shadow and Moustache", "3oclockmoustache"},
+            {"5 O'clock Shadow", "5oclock"},
+            {"5 O'clock Shadow and Moustache", "5oclockmoustache"},
+            {"7 O'clock Shadow", "7oclock"},
+            {"7 O'clock Shadow and Moustache", "7oclockmoustache"},
+            {"Mutton Chops", "mutton"},
+            {"Mutton Chops and Moustache", "muttonmu"},
+            {"Walrus Moustache", "walrus"},
```

Comment:
This feels like it's a bit too hard-coded... Perhaps it could be changed so those can be defined in YAML or something?

Reply:
Yes but what's to be gained?

Reply:
Not having to recompile the game each time someone adds a new hairstyle, for example

Reply:
Then again, it's not a big deal so whatever, really